### PR TITLE
Update cover.rflink.markdown

### PR DESCRIPTION
**Pull request by fmartens**
**Description:**

This documentation fix is describing a property added to resolve #10810
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by fmartens** [comment]
The Home Assistant pull request: https://github.com/home-assistant/home-assistant/pull/22842

File: source/_components/cover.rflink.markdown [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
 title: "RFLink Cover"
-description: "Instructions on how to integrate RFLink Somfy RTS Cover into Home Assistant."
-date: 2017-08-08
+description: "Instructions on how to integrate RFLink Somfy RTS and KAKU ASUN-650 covers into Home Assistant."
+date: 2019-04-07
```
Comment (klaasnicolaas) [inline source/_components/cover.rflink.markdown:5]: Please do not change the `date`.

Comment (fmartens) [reply source/_components/cover.rflink.markdown:5]: @klaasnicolaas , My apologies, i did not know that.

**Comment by fmartens** [comment]
We can close this pull request. I have been asked to rewrite the implementation. The new implementation will not match the changes I made to the documentation.

**Pull request closed without merging**